### PR TITLE
Rustup

### PR DESCRIPTION
**Pull request by tamird**
@alexcrichton there's a failing test here (also, could you set up travis for this repo?) that I wasn't sure how to address so just marked it for now

### CODE REVIEW

**Comment by IvanUkhov** [comment]
@tamird, there are two more lonely Copies [here](https://github.com/tamird/hamcrest-rust/blob/master/src/hamcrest/matchers/existing_path.rs#L7-L14). Do you mind updating the pull request? Thank you.

File: src/hamcrest/matchers/existing_path.rs [hunk 1]
```diff
@@ -4,14 +4,14 @@ use std::fs;
 
 use {success, expect, Matcher, MatchResult};
 
-#[derive(Copy)]
+#[derive(Clone, Copy)]
```
Comment (tamird) [inline src/hamcrest/matchers/existing_path.rs]: @IvanUkhov ... ?

Comment (IvanUkhov) [inline src/hamcrest/matchers/existing_path.rs]: @tamird, sorry, I didn’t notice that it was a different branch. Looks good. Thanks!)

**Comment by alexcrichton** [comment]
I don't have access to enable travis for this repo, I've just been updating it for Cargo itself (I also haven't run the tests in quite awhile clearly!).

For now could the last commit be left out perhaps? The failure can be investigated later of course.

**Comment by tamird** [comment]
@alexcrichton done

**Comment by alexcrichton** [comment]
Thanks!

**Comment by tamird** [comment]
@carllerche would you mind setting up Travis-CI on this repo? the `.travis.yml` file has already been added.

**Pull request merged**